common/lib/capa/capa/tests/helpers.py

```diff
@@ -116,6 +117,6 @@ def load_fixture(relpath):
     in the same directory as the test file.
     """
     abspath = os.path.join(os.path.dirname(__file__), 'test_files', relpath)
-    with open(abspath) as fixture_file:
+    with io.open(abspath) as fixture_file:
```

Comment:
Might want to specify the encoding here in case the system default isn't utf8, but otherwise 👍